.golangci.yml

```diff
@@ -41,3 +42,7 @@ issues:
     - linters:
         - goheader
       text: "go-ethereum Authors" ## disable check for other authors
+    - linters:
+        - forbidigo
+      path: cmd/bee/cmd
+      text: "use of `fmt.Print"   ## allow fmt.Print in cmd directory
```

Comment:
Do we need the "`"?

Reply:
yes, because exact text of lint error could be:

```"use of `fmt.Printf` forbidden by pattern"```,
```"use of `fmt.Println` forbidden by pattern"```, etc